### ISSUE TITLE
Seneca 3.0.0 fails at creating a new instance of seneca

### ISSUE THREAD

**Issue by Bernardstanislas** [senecajs/seneca#522]
When I try to create a new instance, it fails:

```js
var seneca = require('seneca')();
```

produces:

```bash
/home/stan/Documents/nodeschool/seneca/node_modules/seneca/lib/optioner.js:81
      Legacy.parse_command_line(argv.seneca.log,
             ^

TypeError: Legacy.parse_command_line is not a function
    at module.exports (/home/stan/Documents/nodeschool/seneca/node_modules/seneca/lib/optioner.js:81:14)
    at make_seneca (/home/stan/Documents/nodeschool/seneca/node_modules/seneca/seneca.js:253:23)
    at init (/home/stan/Documents/nodeschool/seneca/node_modules/seneca/seneca.js:184:16)
    at Object.<anonymous> (/home/stan/Documents/nodeschool/seneca/exo1-sum.js:1:95)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
```

I tried using node `0.12.4`, `4.5.0`, `6.2.0`, all would fail.

**Comment by tswaters** [comment]
I've encountered this myself, relates to using `--seneca.log` via the CLI which [according to the docs](http://senecajs.org/docs/tutorials/logging-with-seneca.html) is still valid.

Even after installing `seneca-legacy-logger` and changing instantiation like so:

```
require('seneca')({internal: {logger: require('seneca-legacy-logger')}})
```

Still fails.... is there a way to pass log levels with 3.0?  What is it?

**Comment by mihaidma** [comment]
Related to: https://github.com/senecajs/seneca/issues/523
You can see in that post links to sample loggers and sample API usage. Especially look at [Log level API samples](https://github.com/senecajs/senecajs.org/blob/master/src/pages/api/index.md) for log level explanations.

[Logging with seneca](http://senecajs.org/docs/tutorials/logging-with-seneca.html) is updated now, today a new version of it will be published.

The `--seneca.log` options are no longer working with latest log implementation.

**Comment by mcdonnelldean** [comment]
@Bernardstanislas @tswaters We are adding back in this functionality in a more generic way. Basically we will again parse these flags and then pass them to each logger.

Loggers don't have to follow them (or we don't enforce it) but we will provide a reusable module that filters logs based on seneca 2.x

Should all be ready by Monday. The net effect is both old and new loggers will have command line handling back.

**Comment by mcdonnelldean** [comment]
Merged with #516 Same issue, also related to #536 Which is set to fix it.
